ga4gh/frontend.py

```diff
@@ -32,6 +33,7 @@
 SECRET_KEY_LENGTH = 24
 
 app = flask.Flask(__name__)
```

Comment:
Just in the interest of paranoia, can we put in an `assert not hasattr(app, 'urls')` here? This is worryingly close to something Flask might use itself, so I don't want to clobber this, now or in the future.